requirements.txt

```diff
@@ -1 +1 @@
-Theano==0.7
+git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
```

Comment:
Is this necessary? I don't think we should require a git version of Theano for our imminent release. We should ensure everything works with 0.7.